app/components/Time/Time.spec.js

```diff
@@ -15,7 +21,7 @@ describe('<Time />', () => {
   it('should show a date with a custom format', () => {
     const dateTime = '2016-02-02T22:17:21.838103Z';
     const wrapper = shallow(<Time format={'HH:mm'} time={dateTime} />);
-    const expected = <time dateTime={dateTime}>23:17</time>;
+    const expected = <time dateTime={dateTime}>22:17</time>;
```

Comment:
haha, I specifically picked this time to make sure we tested the midnight crossover with +1 timezones :P

Reply:
I assume you changed it because it was acting up, is there a real issue here we should be testing?

Reply:
@odinuge

Reply:
:+1: Got tired of dealing with timezones. Will look at this with @finnss today! :smile:

Reply:
no worries, was just trigger happy on the merge button :p